README.md

```diff
@@ -228,6 +228,15 @@ help() {
 
 Then you can do `$ help cp` or `$ help git commit`.
 
+Alternatively, you can also use global aliases to override `-h` and `--help` entirely:
+
+```bash
+alias -g -- -h='-h | bat --language=help --style=plain'
```

Comment:
`alias -g` doesn't seem to be a valid option for me on Linux...

> `bash: alias: -g: invalid option
alias: usage: alias [-p] [name[=value] ... ]`

Reply:
`alias` is a shell command. The example here is probably zsh-specific?

I'm a zsh user, but I didn't know I could do that. That is pretty cool!

@chrisgrieser We should document that this is zsh-only. Are there any cases where this can fail?

To make this less "invasive", one could also use sth like `--bathelp`
```
alias -g -- --bathelp='--help | bat --language=help --style=plain'
```
or `--chelp` (colored help)
```
alias -g -- --chelp='--help | bat --language=help --style=plain'
```

Reply:
yes there are. if a process writes its help text to stderr (`xxd`, for example).

We could use
```zsh
alias -g -- -h='-h 2>&1 | bat --language=help --style=plain'
```

Reply:
all fixed!